win/PyTango_VS9/PyTango.vcproj

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="Windows-1252"?>
 <VisualStudioProject
 	ProjectType="Visual C++"
-	Version="9.00"
+	Version="9,00"

```

Comment:
In fact, both are working. Maybe it's my french version of Visual that changed it. So we can keep the .